core/gesture.ts

```diff
@@ -265,27 +239,30 @@ export class Gesture {
    * @returns True if a block is being dragged from the flyout.
    */
   private updateIsDraggingFromFlyout_(): boolean {
-    if (!this.targetBlock_) {
+    if (!this.targetBlock_ ||
+        !this.flyout_?.isBlockCreatable(this.targetBlock_)) {
       return false;
     }
-    if (!this.flyout_.isBlockCreatable(this.targetBlock_)) {
-      return false;
+    if (!this.flyout_) {
```

Comment:
I think you don't need this check anymore. If flyout is undefined, it'll short-circuit on 243 on return false.